sdk/containerregistry/container-registry/samples/v1-beta/javascript/containerRegistryClient.js

```diff
@@ -59,3 +58,5 @@ async function deleteRepository(client, repositoryName) {
 main().catch((err) => {
   console.error("The sample encountered an error:", err);
 });
+
+module.exports = { main };
```

Comment:
You can either keep this or remove the `export` from the main function if you don't need it. The sample runner doesn't require it anymore.

Reply:
Removed